content/influxdb/v2.0/query-data/execute-queries/influx-query.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Use the `influx query` command
+weight: 204
+menu:
+  influxdb_2_0:
+    name: Use the `influx query` command
```

Comment:
- Don't use inline code formatting in menu item names.
- You could shorten this to "Use the influx CLI" since this only appears in the left nav.